src/plat/imx93/config.cmake

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright 2024, Indan Zupancic
+#
+# SPDX-License-Identifier: GPL-2.0-only
+#
+
+cmake_minimum_required(VERSION 3.7.2)
+
+declare_platform(imx93 KernelPlatformIMX93 PLAT_IMX93 KernelArchARM)
+
+if(KernelPlatformIMX93)
+    declare_seL4_arch(aarch64)
+    set(KernelArmCortexA55 ON)
+    set(KernelArchArmV8a ON)
+    set(KernelArmGicV3 ON)
+    config_set(KernelARMPlatform ARM_PLAT ${KernelPlatform})
+    set(KernelArmVtimerUpdateVOffset OFF)
+    set(KernelArmDisableWFIWFETraps ON)
```

Comment:
These are specific to the deployment of a system using the platform rather than something all users of the platform should have right?

Reply:
It looks like these two are from the setup of `tqma8xqp1gb` that also has then set to the same defaults. But I agree, unless they are necessary for the platform, they should be project settings, not platform settings.

Reply:
Yea I thought they had just been copy and pasted.

They look like project settings to me.

Reply:
Exactly what happened. But I think `KernelArmVtimerUpdateVOffset` at least should be default on, otherwise timing will be messed up in the VM. `KernelArmDisableWFIWFETraps` should stay off by default. But agreed that the default shouldn't be determined by platform settings.